app/models/card.rb

```diff
@@ -6,10 +6,15 @@ class Card < ActiveRecord::Base
 
   after_validation :days3
 
+  scope :cards_older_today, -> { where("review_date < ?", Date.today)}
```

Comment:
Space missing inside }.<br>Do not use `Date.today` without zone. Use `Time.zone.today` instead.